test/e_chronicler/models/journal_entry_test.exs

```diff
@@ -66,5 +67,10 @@ defmodule EChronicler.Models.JournalEntryTest  do
     refute invalid_changeset.valid?
   end
 
+  test "change_journal_entry/1 returns a journal_entry changeset" do
+    assert {:ok, %JournalEntry{} = journal_entry} = EChronicler.Models.JournalEntry.create_journal_entry(@valid_journal_entry)
+
+    assert %Ecto.Changeset{} = JournalEntry.change_journal_entry(journal_entry)
+  end
 
```

Comment:
Where are the tests for `update_journal_entry`?

Reply:
Added them!